roles/ee_builder/tasks/00_build_ee.yml

```
@@ -70,7 +70,7 @@
     tag: "{{ __execution_environment_definition.tag | default(omit) }}"
     validate_certs: "{{ ee_validate_certs | default(omit) }}"
     push_args:
-      dest: "{{ ee_registry_dest }}"
+      dest: "{{ ee_registry_dest }}/{{ __execution_environment_definition.name }}{{ ':' + __execution_environment_definition.tag if __execution_environment_definition.tag else '' }}"
```

Just want to check this is correct if no tag is applied. Will this not leave a trailing `:` with no tag which I assume will fail? Should it not be as below:

```suggestion
      dest: "{{ ee_registry_dest }}/{{ __execution_environment_definition.name }}{{ (':' + __execution_environment_definition.tag) if __execution_environment_definition.tag else '' }}"
```

---

I've not tested so it also could be way it gets evaluated the brackets don't matter, although this may be more obvious what's happening